packages/react/src/alert-dialog/popup/AlertDialogPopup.tsx

```diff
@@ -63,9 +63,10 @@ const AlertDialogPopup = React.forwardRef(function AlertDialogPopup(
   useOpenChangeComplete({
     open,
     ref: popupRef,
-    change: 'open',
     onComplete() {
```

Comment:
The most straightforward and idiomatic way to use this would be to have a parameter on the `onComplete` callback that determines if the popup was opened or closed.

Reply:
I thought of that, but it's not actually necessary because you can just use the outer variable that's passed as `open`. The `useEventCallback` ensures it's not stale.